core/amber/src/main/python/core/architecture/handlers/actorcommand/backpressure_handler.py

```diff
@@ -29,7 +33,12 @@ def __call__(
                     tag=ActorVirtualIdentity(""),
                     payload=set_one_of(
                         ControlPayloadV2,
-                        ControlInvocationV2(-1, set_one_of(ControlCommandV2, NoOpV2())),
+                        ControlInvocation(
+                            "ResumeWorker",  # use resume worker to simulate NoOp
+                            set_one_of(ControlRequest, EmptyRequest()),
+                            AsyncRpcContext(),
+                            -1,
+                        ),
```

Comment:
I thought we are calling overloaded function to resumeWorker, instead of using string parameter. This is still a "ResumeWorker" string, which I think might be a bad design.

Reply:
I agreed but this is a special case where we are generating a control invocation in the `network_receiver`. There is no rpc_client we can use and this is an `ActorCommand` handler.